pkg/api/openshiftcluster.go

```diff
@@ -198,6 +198,7 @@ type NetworkProfile struct {
 
 	PodCIDR     string `json:"podCidr,omitempty"`
 	ServiceCIDR string `json:"serviceCidr,omitempty"`
+	NetworkType string `json:"networkType,omitempty"`
```

Comment:
name we agreed was `SDNProvider`, why changing? NetworkType is ambiguous. 

https://github.com/Azure/ARO-RP/pull/1200/files#diff-32cee431764117c8a02c5eabb82bbf3fa57167d1297f68164cdf154269b8c74aR113

Reply:
> name we agreed

where was that discussion held? I don't think I was a part of that discussion? 

`SDNProvider` is not the name used in the installer, why change the name to something new? 

I agree it isn't the best name, but if we want to change it we should open a PR upstream to installer and request a change there first. We should use the same name, and same JSON location, as the installer expects to maintain consistency in APIs and reduce confusion.

I vote to remain with `NetworkType` to continue to match the current installer implementation.

Reply:
This was like 9 months ago in arch meeting. 
Installer API is not a reference for us as its not RestAPI and naming there is scapfolded from MachineAPI, installer team opinions, cloud provided code base and kubernetes object namings with no consistency towards being rest api.
NetworkType is wrong overloaded term, one example bellow (and there is way more Types out there). So when it lands into APi it becomes confusing even more. 
```
LAN(Local Area Network)
PAN(Personal Area Network)
MAN(Metropolitan Area Network)
WAN(Wide Area Network)
```

Reply:
> I agree it isn't the best name, but if we want to change it we should open a PR upstream to installer and request a change there first.

I dont think its our position to do so and installer team will never accepts this as it will be breaking change for them. Just because somebody named fields wrong, we dont need to blindly follow them.

Reply:
Changed as requested.